loopchain/blockchain/blockchain.py

```diff
@@ -340,6 +383,17 @@ def __write_block_data(self, block: Block, confirm_info):
             block.header.height.to_bytes(conf.BLOCK_HEIGHT_BYTES_LEN, byteorder='big'),
             block_hash_encoded)
 
+        if receipts:
+            batch = self.__add_tx_to_block_db(block, receipts, batch)
+
+        if next_prep:
+            utils.logger.notice(f"store next_prep in __write_block_data\nprep_hash({next_prep['rootHash']})"
+                                f"\npreps({next_prep['preps']})")
+            batch.put(
+                BlockChain.PREPS_KEY + next_prep['rootHash'].encode(encoding=conf.HASH_KEY_ENCODING),
+                json.dumps(next_prep['preps']).encode(encoding=conf.PEER_DATA_ENCODING)
+            )
+
```

Comment:
Is there any information about `startTermHeight` and `endTermHeight`?

Reply:
It's not possible support now.